platform/tests/UNITTESTS/doubles/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@
 add_library(mbed-headers-platform INTERFACE)
 target_include_directories(mbed-headers-platform
     INTERFACE
+        .
+        platform
+        platform/cxxsupport
         ${mbed-os_SOURCE_DIR}/platform/include
```

Comment:
In commit "Unit tests: Move platform target_h stubs", we moved many files to `platform/tests/UNITTESTS/doubles/platform`

But we moved `nvic_wrapper.h` to `platform/tests/UNITTESTS/doubles/`. Was this inconsistency intentional? Do we need the `platform/tests/UNITTESTS/doubles/platform` folder still, rather than putting everything under `platform/tests/UNITTESTS/doubles/`? We ought to know we are in the platform folder and this is redundant, but it would depend if any files were included like `#include "platform/CThunk.h` by mentioning the folder. Does nobody depend on `nvic_wrapper.h` this way and that's why we could move it out of a folder named `platform`?

Reply:
Yes, `#include "platform/CThunk.h` is the preferred style of includes. Even though it's not strictly followed by some code, in my latest push I just moved `nvic_wrapper.h` into `platform/tests/UNITTESTS/doubles/platform` for consistency.